portal-web/docroot/html/taglib/ui/page_iterator/start.jsp

```diff
@@ -410,6 +414,77 @@ NumberFormat numberFormat = NumberFormat.getNumberInstance(locale);
 	}
 </script>
 
+<script data-senna-track="temporary" type="text/javascript">
```

Comment:
This script tag should be moved inside the same <#if>  block that the pagination dropdown is inside of. We want to make sure this JS is only rendered if needed.

Reply:
Moved to the bottom of the <#if> block.
Thanks for the review @ethib137!